pkg/handler/popular_repositories.json

```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "suggested_name": "Epel-9-Everything-x86_64",
```

Comment:
I'd probably change these names to   "EPEL 9 Everything x86_64"

I do wonder if we should go with the full "Extra Packages for Enterprise Linux 9 Everything x86_64", but that seems too long :)

Reply:
Added the first for now.